tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-step-selector.html

```diff
@@ -0,0 +1,148 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-slider/paper-slider.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-card-heading/tf-card-heading.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-imports/lodash.html">
+
+<!--
+  Renders a UI for selecting a step.
+-->
+<dom-module id="tf-step-selector">
+  <template>
+    <div id="step-display-container">
+      step <span id="step-text">[[_stepText]]</span>
+    </div>
+    <div id="run-display-container">
+      <div id="run-color-box" style="background:[[color]];"></div>
+      <div id="run-text">
+        [[run]]
+      </div>
+    </div>
+    <paper-slider
+        id="slider"
+        value="{{_stepIndex}}"
+        immediate-value="{{_stepIndex}}"
+        step="1"
+        type="number"
+        min=0
+        max="[[_maxStepIndex]]"
+        ></paper-slider>
+    <style>
+      #slider {
+        margin: -8px 0 0 -15px;
+        width: calc(100% + 31px);
+      }
+      
+      #step-display-container {
+        margin: 10px 15px 0 0;
+      }
+
+      #run-display-container {
+        font-size: 0.7em;
```

Comment:
It doesn't feel right to have the step value larger than the run name. I can already see the approximate step from the slider; I'm more likely to be scanning for a particular run to change. Suggest equal sizes or run larger.